example/javascriptlib/basic/6-client-server-realistic/build.mill

```diff
@@ -0,0 +1,68 @@
+package build
+import mill._, javascriptlib._
+
+object client extends ReactScriptsModule
+
+object server extends JestModule {
+  def npmDeps =
+    Seq("@types/cors@^2.8.17", "@types/express@^5.0.0", "cors@^2.8.5", "express@^4.21.1")
+
+  def npmDevDeps =
+    super.npmDevDeps() ++ Seq("@types/supertest@^6.0.2", "supertest@^7.0.0")
+
+  override def testConfigSource = Task.Source(millSourcePath / "jest.config.ts")
+
+  override def bundleFlags = Map("external" -> Seq("express", "cors"))
+
+  override def mkENV = Task {
+    super.mkENV() ++ Map("PORT" -> "3001")
+  }
+
+}
+
+object app extends RsWithExpressModule {
+  def clientMod = client
+  def serverMod = server
+}
+
+// Documentation for mill.example.javascriptlib
+
+// This example demonstrates how to set up a simple React web app running on an Express JS server
+// implementing the popular Todo-MVC demo application. It includes a test suite
+// for both client and server modules.
+// Runninng `mill app.run` serves bundled static HTML files through the server.
+
+/** Usage
```

Comment:
Similar to above, let's trim the output line assertions. Just 1-2 lines each should be enough to demonstrate what is important and ensure the right logic is mostly being run